generator/github_template/github-workflow.yml

```diff
@@ -0,0 +1,24 @@
+name: Workflow
+
+# trigger when new commits are pushed to master or main branch
+on:
+  push:
+    branches: [ master, main ]
```

Comment:
`main` branch seems real specific and not there when you use the plugin. I suggest to just trigger on `master`